.github/workflows/additional-ci-image-checks.yml

```diff
@@ -142,7 +140,7 @@ jobs:
       - name: "Login to ghcr.io"
         run: echo "${{ env.GITHUB_TOKEN }}" | docker login ghcr.io -u ${{ github.actor }} --password-stdin
       - name: "Check that image builds quickly"
-        run: breeze shell --max-time 120
+        run: breeze shell --max-time 600 --platform "linux/amd64"
```

Comment:
Is `600`s (10min) timeout intentionally a minute lower than the 11 minute one above?

Reply:
Yes. there are few steps above that take usually ~ 1 minute :)

Reply:
(quite a bit less than that usually)